Gruntfile.js

```diff
@@ -355,19 +361,165 @@ module.exports = function (grunt) {
 							]
 						},
 						{
-							config: 'bump.version',
+							// if custom bump is used with a specific version, see dorelease task
+							config: 'release.buildSemVerType',
 							type: 'input',
 							message: 'What specific version would you like',
 							when: function (answers) {
-								return answers['bump.increment'] === 'custom';
+								return answers['release.buildSemVerType'] === 'custom';
 							},
 							validate: function (value) {
 								var valid = semver.valid(value);
 								return valid || 'Must be a valid semver, such as 1.2.3-rc1. See http://semver.org/ for more details.';
 							}
+						},
+						{
+							config: 'release.remoteRepository',
+							default : 'upstream',
```

Comment:
What I'm saying is, why not use `grunt.config.release.remoteRepository` here instead of the string "upstream"?

So:
`default: grunt.config.release.remoteRepository,`